db_schema_diagram.sql

```diff
@@ -0,0 +1,70 @@
+CREATE DATABASE clinic;
+CREATE TABLE patients (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(100) NOT NULL,
+  date_of_birth DATE NOT NULL
+);
+
+CREATE TABLE medical_histories (
+  id SERIAL PRIMARY KEY,
+  admitted_at TIMESTAMP,
+  patient_id INT NOT NULL,
+  status VARCHAR(150)
+);
+CREATE TABLE treatments (
+    id SERIAL PRIMARY KEY,
+    type VARCHAR(255),
+    NAME VARCHAR(255)
+);
+-- invoices table
+CREATE TABLE invoices (
+    id SERIAL PRIMARY KEY,
+    total_amount DECIMAL,
+    generated_at TIMESTAMP,
+    paid_at TIMESTAMP,
+    medical_history_id INTEGER
+);
```

Comment:
- [ ] As it was raised earlier, kindly create an index for this foreign key: **medical_history_id INTEGER.**

`CREATE INDEX ON invoices (medical_history_id);`

[optional] You did not set it the **id** as foreign key, I implore you to implement it. The same is applicable to `medical_histories` table as well. 👍